src/transformers/models/whisper/tokenization_whisper_fast.py

```diff
@@ -312,6 +312,12 @@ def decode(
         return text
 
     def _decode(self, *args, normalize: bool = False, **kwargs) -> str:
+        token_ids = kwargs["token_ids"]
+        skip_special_tokens = kwargs["skip_special_tokens"]
+        has_prompt = isinstance(token_ids, list) and token_ids[0] == self.convert_tokens_to_ids("<|startofprev|>")
+        if skip_special_tokens and has_prompt:
```

Comment:
Same here. I'd prefer to do the `has_prompt` check after checking for `skip_special_tokens`. (It's only a small thing and `skip_special_tokens` will be True most of the time anyway, so consider this a nitpick. ;-) )